source/beamline/SXRMB/SXRMBBeamline.cpp

```diff
@@ -642,19 +642,22 @@ void SXRMBBeamline::setupSampleStage()
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageZ_);
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageR_);
 
-	// Ambiant Endstation sample state
+	// Ambiant Endstation sample stage
+	ambiantTableHeight_ = new AMPVwStatusControl("Ambiant Table Height", "TBL0000-E07-01:Y:mm:fbk", "TBL0000-E07-01:Y:mm", "TBL0000-E07-01:Y:mm:status", QString(), this, 0.005, 2.0, new CLSMAXvControlStatusChecker());
```

Comment:
@XuetaoLiu-CLS, all motors have stop PV's.  Although it may not be visible at the layer you are looking at, there will be a stop PV at the MAXv motor level driver layer.  If anything, if it doesn't exist, talk to @beaured0 about setting one up.